src/bots/ama/subCommands/listEvents.js

```diff
@@ -10,50 +11,74 @@ const handler = async () => {
   if (events.length === 0) {
     await client.message.channel.send(getStrings().noEvents);
   } else {
-    client.message.channel.send(formatEvents(events));
+    let parsedCmd = parseCommandString();
+
+    client.message.channel.send(
+      formatEvents(events, parsedCmd.arguments.length > 0)
+    );
   }
 };
 
 /**
  * Formats events into a readable string
  * @param {Array.<Object.<string, any>>} events - list of event objects
+ * @param {boolean} showIds - trigger that shows event Database IDs if set to true
  * @returns {string} - message string with all events
  */
-const formatEvents = (events) => {
+const formatEvents = (events, showIds) => {
   let currentDate = new Date();
   let splitDate = partition(events, (e) => e.date > currentDate);
+  let eventString = '';
+
+  if (splitDate[1].length > 0) {
+    let pastEvents = '';
 
-  let upcomingEvent = getStrings(
-    `${splitDate[0][0].date.toDateString()}: ${splitDate[0][0].title}`
-  ).upcomingEvent;
+    for (let s in splitDate[1]) {
+      let next = splitDate[1][s];
 
-  let futureEvents = '';
+      pastEvents += `${
+        showIds ? next.id + ' ' : ''
+      }${next.date.toDateString()}: ${next.title}\n`;
+    }
 
-  for (let i = 1; i < splitDate[0].length; i++) {
-    let next = splitDate[0][i];
+    pastEvents = getStrings(pastEvents).pastEvents;
 
-    futureEvents += `${next.date.toDateString()}: ${next.title}\n`;
+    eventString += pastEvents + '\n';
   }
 
-  futureEvents = getStrings(futureEvents).futureEvents;
+  if (splitDate[0].length > 0) {
+    let upcomingEvent = getStrings(
+      `${
+        showIds ? splitDate[0][0].id + ' ' : ''
+      }${splitDate[0][0].date.toDateString()}: ${splitDate[0][0].title}`
```

Comment:
for all the references to the array such as `splitDate[0]` or `splitDate[1]` or `splitDate[0][0]`, can we store them in variables at the top of the function so that

1. you only have to access that once
2. it's clearer what each of these refer to
3. it's more readable